addons/l10n_se/data/account_account_template.xml

```diff
@@ -0,0 +1,1973 @@
+            <field name="name">Periodiseringsfond 2016</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2127" model="account.account.template">
+            <field name="code">2127</field>
+            <field name="name">Periodiseringsfond 2017</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2128" model="account.account.template">
+            <field name="code">2128</field>
+            <field name="name">Periodiseringsfond 2018</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2129" model="account.account.template">
+            <field name="code">2129</field>
+            <field name="name">Periodiseringsfond 2019</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2130" model="account.account.template">
+            <field name="code">2130</field>
+            <field name="name">Periodiseringsfond 2020</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2131" model="account.account.template">
+            <field name="code">2131</field>
+            <field name="name">Periodiseringsfond 2021</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2132" model="account.account.template">
+            <field name="code">2132</field>
+            <field name="name">Periodiseringsfond 2022</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2133" model="account.account.template">
+            <field name="code">2133</field>
+            <field name="name">Periodiseringsfond 2023</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
+        <record id="coa_2134" model="account.account.template">
+            <field name="code">2134</field>
+            <field name="name">Periodiseringsfond 2024</field>
+            <field name="user_type_id" ref="account.data_account_type_equity"/>
+            <field name="chart_template_id" ref="se_chart_template"/>
+        </record>
```

Comment:
@daniel-stenlov Hello

"Periodiseringsfond" account is repeated with the year it's required? 
Because we filter entries by a year

Reply:
Yes. Periodiseringsfond is used to move taxes from one year to another. You are allowed to put 30% of profit into a periodiseringsfond for a year, if liked and you can move it back into company at will. So basicly you can lower taxes with 30% and move it back in a year when you don't have a profit. However you need to keep track of what year it's from and you need to report changes to the tax authority.

Reply:
@daniel-stenlov ok, a good reason do this 👍